webapp/components/survey/NodeDefDetails/store/actions/useSaveEdits.js

```diff
@@ -47,7 +47,7 @@ export const useSaveEdits = ({ setState }) => {
       let chainNodeDef = null
       if (NodeDef.isAnalysis(nodeDefUpdated)) {
         chainNodeDef = {
-          chainUuid: chain.uuid,
+          chainUuid: chain?.uuid,
```

Comment:
chain must be there if adding a nodeDef analysis. add a check if it's an attribute instead .